sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -269,6 +269,8 @@ def __init__(self, packages, options, environment_version, pipeline_url):
     if self.standard_options.streaming:
       # Use separate data disk for streaming.
       disk = dataflow.Disk()
+      if self.worker_options.disk_type:
```

Comment:
This should probably go after the 'local' part, so that changing the type is always possible if desired